layouts/partials/challenge.html

```diff
@@ -0,0 +1,16 @@
+<div  id="challenge" class="content-section">
+  <div class="container text-left">
+    <div class="col-md-8 col-md-offset-2">
+
+      <h2>Legal Challenge to SESTA/FOSTA:<br/>The Woodhill Case</h2>
+      <p>In Woodhull Freedom Foundation et al. v. United States, a wide variety of plaintiffs are asking the court to declare SESTA/FOSTA unconstitutional: two human rights organizations, a digital library, a sex work activist, and a certified massage therapist.</p>
+      <p>SESTA/FOSTA violates the First Amendment by punishing certain types of lawful speech – including speech advocating for the decriminalization of sex work. Its broad wording creates a great deal of uncertainty, both for Internet users and for online platforms about what speech or activity creates liability under the law.</p>
+      <p>The law also violates the Constitution’s prohibition against ex post facto laws—laws that retroactively criminalize activity that was legal at the time that it happened. </p>
+      <p><em>[insert links to newest blog posts on Woodhull]</em></p>
+      <div class="action-button button">
+        <a href="https://eff.org" ><span class="larger">Read More</span><br/>on eff.org
```

Comment:
should this link to `https://www.eff.org/cases/woodhull-freedom-foundation-et-al-v-united-states`?